python/sglang/check_env.py

```diff
@@ -9,6 +9,19 @@
 
 import torch
 
+is_rocm = False
```

Comment:
use this one?
https://github.com/sgl-project/sglang/blob/ece724910afd624e8aac85444472440d6586c3e4/python/sglang/srt/utils.py#L67-L69